### ISSUE TITLE
Models don't perform as expected when trained on a large train set

### ISSUE THREAD

**Issue by atkm** [atkm/avazu-ctr#4]
From Kaggle submissions (private scores):

Model 3:
- Half of train.csv: 0.4241803
- train_small.csv: 0.4123864

Model 1:
- Half of train.csv: 0.4172254
- train_small.csv: 0.4168748

Run an experiment to pick an optimal training set size.
One way to reduce the training set is to remove rows with (device_id, site/app_id) pairs that do not appear on the test set.

**Comment by atkm** [comment]
Maybe the model needs a different regularization parameter when fitting to a large dataset?

**Comment by atkm** [comment]
A simple classifier like logistic regression shouldn't require as much data as a more complex one like decision trees.
